src/components/Link/Link.tsx

```diff
@@ -33,17 +37,17 @@ type LinkProps = {
  * A link component.
  */
 export const Link = forwardRef<HTMLAnchorElement, PropsWithChildren<LinkProps>>(
```

Comment:
That typing is wrong. If you make it a `button`, the ref should be an HTMLButtonElement. You would need to do something similar of what has been done for [the button component](https://github.com/vector-im/compound-web/blob/af199c604e444cf4d49b15e303c1f121b85ee8a3/src/components/Button/Button.tsx#L27)